src/Common/src/Interop/Windows/kernel32/Interop.GlobalUnlock.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, PreserveSig = true)]
+        internal static extern int GlobalUnlock([In] IntPtr hMem);
+    }
+}
```

Comment:
Splitting GlobalLock and GlobalUnlock into multiple files is ok, but FWIW, I think it'd be fine to have GlobalUnlock in the same file as GlobalLock: I can't think of a situation where you'd want one without the other.